### PR TITLE
document kubectl delete --interactive flag

### PR DESCRIPTION
**Pull request by ardaguclu**
This PR adds document for the alpha stage of https://github.com/kubernetes/enhancements/issues/3895

### CODE REVIEW

**Review by soltysh — APPROVED** [review]
/lgtm
from tech pov

File: content/en/docs/reference/kubectl/kubectl.md [hunk 1]
```diff
@@ -369,6 +369,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_INTERACTIVE_DELETE</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, --interactive flag in kubectl delete command will be activated and by passing this flag, users can preview and confirm resources before proceeding to delete.
```
Comment (chrisnegus) [inline content/en/docs/reference/kubectl/kubectl.md:376]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the --interactive flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete. 
```

**Review by chrisnegus — COMMENTED** [review]
A small wording suggestion.

File: content/en/docs/reference/kubectl/kubectl.md (hunk 1 above)
Comment (ardaguclu) [reply content/en/docs/reference/kubectl/kubectl.md]: Thanks @chrisnegus for the suggestion. Updated.

File: content/en/docs/reference/kubectl/kubectl.md [hunk 2]
```diff
@@ -369,6 +369,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_INTERACTIVE_DELETE</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the --interactive flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete by passing this flag.
```
Comment (sftim) [inline content/en/docs/reference/kubectl/kubectl.md:376]: <td></td><td style="line-height: 130%; word-wrap: break-word;">When set to true, the <tt>--interactive</tt> flag in the kubectl delete command will be activated, allowing users to preview and confirm resources before proceeding to delete by passing this flag.

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

Fixup PRs welcome!

**Pull request merged**